src/Route/AdminPoolLoader.php

```diff
@@ -63,13 +64,13 @@ public function load($resource, $type = null)
                 $collection->add($route->getDefault('_sonata_name'), $route);
             }
 
-            $reflection = new \ReflectionObject($admin);
+            $reflection = new ReflectionObject($admin);
```

Comment:
@sonata-project/contributors Should decide a CS rule about this ?

Reply:
It was suggested by PHPStorm, that is why I adjusted it.
Should I change it back?

Reply:
I have no opinion on this.

But I would recommend to revert it indeed
- to avoid potential conflict
- avoid delaying this PR because of a discussion about this change
- it's better to have this change made automatically

I looked if Phpcs https://github.com/FriendsOfPHP/PHP-CS-Fixer provides a rule about this change. And yes:
Seems like it's the `global_namespace_import` rule.

If we update our config (https://github.com/sonata-project/dev-kit/blob/master/templates/project/.php_cs.dist) we could then have this change made automatically on every repository and it would give us a real consistency.

Do you want to try a PR on dev-kit ? ;)

Reply:
Nevermind I opened the PR https://github.com/sonata-project/dev-kit/pull/1170 @will2877